pootle/apps/pootle_fs/files.py

```diff
@@ -205,13 +205,18 @@ def _sync_from_pootle(self):
             f.write(self.serialize())
         logger.debug("Pushed file: %s" % self.path)
 
-    def _sync_to_pootle(self, merge=False, user=None):
+    def _sync_to_pootle(self, merge=False, user=None, pootle_wins=None):
```

Comment:
k, but it'd be helpful to see some real cases. I assume you had some reasons when added `pootle_wins` param.